src/Instance/InstanceMarc/InstanceMarc.js

```diff
@@ -0,0 +1,78 @@
+import React, { useMemo } from 'react';
+import { useIntl } from 'react-intl';
+import PropTypes from 'prop-types';
+
+import {
+  Pane,
+  Paneset,
+  Headline,
+} from '@folio/stripes/components';
+
+import InstanceMarcField from './InstanceMarcField';
+import {
+  isControlField,
+} from './utils';
+import styles from './InstanceMarc.css';
+
+const InstanceMarc = ({
+  instance,
+  marc,
+  onClose,
+}) => {
+  const intl = useIntl();
+
+  const renderFields = useMemo(() => ([
```

Comment:
@NikitaSedyx could you do this processing in: https://github.com/folio-org/ui-inventory/pull/1096/files#diff-17145ff2fb1078fd7b5f31197e808f2cR31 right after you get the response back. Then you wouldn't need to use `useMemo` and actually not even pass the entire `marc` prop to `InstanceMarc` just the fields.